core/src/test/java/org/elasticsearch/index/mapper/BinaryFieldMapperTests.java

```diff
@@ -92,7 +93,7 @@ public void testStoredValue() throws IOException {
         assertTrue(CompressorFactory.isCompressed(new BytesArray(binaryValue2)));
 
         for (byte[] value : Arrays.asList(binaryValue1, binaryValue2)) {
-            ParsedDocument doc = mapper.parse("test", "type", "id", XContentFactory.jsonBuilder().startObject().field("field", value).endObject().bytes());
+            ParsedDocument doc = mapper.parse(SourceToParse.source("test", "type", "id", XContentFactory.jsonBuilder().startObject().field("field", value).endObject().bytes(),XContentType.JSON));
```

Comment:
Reviewed all of the instances and where the line count was seemed too large or appeared difficult to read, it was separated into other lines.